test/helpers.spec.ts

```diff
@@ -281,6 +285,26 @@ describe('helpers', () => {
           expect(subset).toHaveLength(new Set(subset).size);
         });
 
+        it('should return a subset with random elements in the array for a length range', () => {
+          const testArray = ['hello', 'to', 'you', 'my', 'friend'];
+          const subset = faker.helpers.arrayElements(testArray, {
+            min: 2,
+            max: 4,
+          });
+
+          // Check length
+          expect(subset.length).toBeGreaterThanOrEqual(2);
+          expect(subset.length).toBeLessThanOrEqual(4);
+
+          // Check elements
+          subset.forEach((element) => {
+            expect(testArray).toContain(element);
+          });
+
+          // Check uniqueness
+          expect(subset).toHaveLength(new Set(subset).size);
```

Comment:
We have [a custom matcher](https://github.com/faker-js/faker/blob/8bf716225e9c5b3ce5c968511886b9141d5c918d/test/vitest-extensions.ts#L4) for this test use-case which resolves a better error message in case it fails.

```suggestion
          expect(subset).not.toContainDuplicates();
```